azkaban-common/src/main/java/azkaban/executor/ExecutableRamp.java

```diff
@@ -376,6 +376,13 @@ public boolean isNotTestable() {
     return (!this.state.isActive || this.state.isPaused || (this.state.rampStage <= 0));
   }
 
+  /**
+   * Return true iif this ramp will not be required to put those failed workflow into the exclude-ramp list
+   */
+  public boolean isMagneticTest() {
```

Comment:
Shall this method name be changed? Magnetic is very specific to Spark right?

Reply:
what is your suggestion?

Reply:
changed to ignoreTestFailure